### ISSUE TITLE
Consume the API Proxy in SAP Web IDE

### ISSUE THREAD

**Issue by NancyT33333** [SAPDocuments/Tutorials#4457]
Tutorial URL: https://developers.sap.com/tutorials/hcp-apim-webide-int.html

I'm referring to steps 12-13.
I have created destinations in my HCP account as specified in steps 6-7. But I'm still unable to see list of APIs in webide service catalog when creating new project from template. 
What settings can I fix to see the list of APIs in webide?

Destinations parameters are:
`Description=for OpenSAP APImgmt demo
Type=HTTP
TrustAll=true
Authentication=BasicAuthentication
WebIDEUsage=api_mgmt_proxy
Name=APIMgmt_APIPortal
WebIDEEnabled=true
URL=https\://s**********trial-trial.apim1.hanatrial.ondemand.com\:443/
ProxyType=Internet
User=**********
`

`Name=APImgmt
URL=https\://devportalu34f5b50f-s***********trial.hanatrial.ondemand.com/
ProxyType=Internet
Type=HTTP
WebIDEEnabled=true
Authentication=AppToAppSSO
WebIDEUsage=api_mgmt_catalog
Description=for apimgmt OpenSap demo dev portal
`
![SAP Web IDE Full-Stack](https://user-images.githubusercontent.com/25182576/72050979-6f98e200-32d3-11ea-8d24-b6887333a7be.png)
Thanks!

**Comment by shilpavij** [comment]
Hello,
In order to have API Products, you need to create one in API Management and even then if the APIs doesn't come up the there is a problem with dev portal destination. Please check the spaces

Regards
Shilpa

**Comment by NancyT33333** [comment]
Hello, Shilpa
Thanks for your answer!
I have created API Products in API Management and I've logged in to webide and api management from the same s-user account.  In the network tab of WebIDE I can see the same list of my API applications as in Dev portal, so destination is probably configured correctly.

![Network requests in webide](https://user-images.githubusercontent.com/25182576/74217532-c1f56600-4cb8-11ea-93ab-990452bfe089.png)
The same Application in dev portal:
![developer portal](https://user-images.githubusercontent.com/25182576/74217645-1993d180-4cb9-11ea-85e8-189873820b03.png)
API product is also associated with API in API Portal
![API product associated](https://user-images.githubusercontent.com/25182576/74218033-35e43e00-4cba-11ea-889e-01e3e9d19eee.png)

Best regards,
Anna
